src/lib/zodSchema/loginSchema.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from 'zod';
+
+export type LoginSchemaType = typeof LoginSchema;
```

Comment:
Using z.infer to generate the type for the schema fields will be better, I think the schema type itself is not important, you can use ZodSchema type for that which is more generic.

Reply:
True this👍